src/Nethermind/Nethermind.Evm/EvmStack.cs

```diff
@@ -107,9 +107,11 @@ public void PushByte(byte value)
             }
         }
 
+        private static readonly byte[] OneStackItem = {1};
+        
         public void PushOne()
         {
-            if (_tracer.IsTracingInstructions) _tracer.ReportStackPush(new byte[] {1});
+            if (_tracer.IsTracingInstructions) _tracer.ReportStackPush(OneStackItem);
```

Comment:
Can you check how it look in the IL code? I think that we replace a constant (0 operations) with a static variable access.